src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/ServiceOfNotice.cshtml

```diff
@@ -0,0 +1,110 @@
+﻿@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Core.DataStandards
+@using EA.Weee.Web.Areas.Producer.Controllers
+@using MvcNavigationHelpers
+@using EA.Weee.Web.RazorHelpers
+@model EA.Weee.Web.Areas.Producer.ViewModels.ServiceOfNoticeViewModel
+
+@{
+    const string title = "Service of notice";
+    ViewBag.Title = title + " - GOV.UK";
+}
+@section full_main_content
+{
+    @(this.WeeeGds().BackLink(@Url.UrlFor<EA.Weee.Web.Areas.Producer.Controllers.ProducerController>(a => a.TaskList())))
+
+    <span class="govuk-caption-xl govuk-!-padding-bottom-3">Step 3 of 5</span>
```

Comment:
Have a look at how the org screen screen adjusts the steps as without a representing company the steps differ.